### ISSUE TITLE
#-779972327------------- All page content should be contained by landmarks

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1836]
<strong>PageName:</strong> FeedDetailsPage  

<strong>Source:</strong>`<div class="esri-ui-inner-container esri-ui-manual-container">`

<strong>Element Location: </strong>.esri-ui-manual-container

<strong>Impact: </strong>moderate

<strong>Failure Summary:</strong> Fix any of the following:
  Some page content is not contained by landmarks

<strong>Tags:</strong> cat.keyboard,best-practice

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.6/region?application=axeAPI
<strong>Screenshot:<strong>`C:\Automation\Workspace\Velocity\Screenshot\FeedDetailsPage\-779972327_All page content should be contained by landmarks.png`
